contracts/test/forkMainnet/AMMWrapper/CollectFee.t.sol

```diff
@@ -1,62 +1,62 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.7.6;
 
+import "contracts/AMMWrapper.sol";
 import "contracts-test/forkMainnet/AMMWrapper/Setup.t.sol";
 import "contracts-test/utils/BalanceSnapshot.sol";
 
 contract TestAMMWrapperCollectFee is TestAMMWrapper {
     using BalanceSnapshot for BalanceSnapshot.Snapshot;
 
-    function testCollectFeeIfReceivedSameAsMinOut() public {
-        uint256 feeFactor = 5;
+    function testFailedIfMakerAssetAmountNotDeductFee() public {
```

Comment:
IMO `MakerAssetAmountNotDeductFee` is not so straightforward. From contract's point of view, the only reason it fails is because the minimum amount is not high enough after the it adds fee on top of `makerAssetAmount`. It has no idea about whether fee is subtracted from `makerAssetAmount` or not.

Maybe `testCannotCollectFeeIfReceivedLessThanMinOutPlusFee`?